pkg/storaged/nfs-details.jsx

```diff
@@ -44,7 +45,13 @@ function nfs_busy_dialog(client,
             dialog.open({ Title: dialog_title,
                           Teardown: {
                               HasUnits: true,
-                              Units: users.map(function (u) { return { Unit: u.unit, Name: u.desc }; })
+                              Units: users.map(function (u) {
+                                  console.log(u);
```

Comment:
Debugging leftover? If you want to keep this, log `JSON.stringify(u)` so that you actually see the object properties instead of just `[Object]`? If not, please turn this into an arrow function, for less syntactic overhead.

Reply:
Yes, debugging leftover.  Looks like it's gone already...